src/app/analysis-view/analysis-view/analysis-view.component.html

```diff
@@ -0,0 +1,24 @@
+<h1>{{algorithmName}}</h1>
+
+<mat-card *ngFor="let analysis of analyses">
+    <mat-card-title>{{analysis.name}}</mat-card-title>
+    <mat-card-content>
+        <a routerLink="../../results/{{algorithmName}}/{{analysis.id}}">Details</a>
+    </mat-card-content>
+</mat-card>
+
+<mat-card *ngIf="analyses!==undefined && analyses.length===0">
+    <mat-card-title>Nothing to see here yet...</mat-card-title>
+    <mat-card-content>
+        Maybe you will start a new analysis?
+    </mat-card-content>
+</mat-card>
+
+<mat-card *ngIf="errorMessage!==undefined">
+    <mat-card-title>An error occured</mat-card-title>
+    <mat-card-content>
+        {{errorMessage}}
+    </mat-card-content>
+</mat-card>
+
+<a mat-fab routerLink="../../analysis/new/{{algorithmName}}">+</a>
```

Comment:
You may try to put this in mat-action-row

Reply:
`mat-action-row` seems to be related only to expansion panel, so it would enforce us to hide this button. I'd leave it as it is.